api/pkg/transformer/feast/feature_retriever.go

```diff
@@ -3,13 +3,15 @@ package feast
 import (
 	"context"
 	"encoding/base64"
+	"encoding/json"
 	"errors"
 	"fmt"
 	"math"
 	"strings"
 	"time"
 
 	"github.com/afex/hystrix-go/hystrix"
+	"github.com/cespare/xxhash"
```

Comment:
Is there any specific reason for using this library? I thought built in library such as https://golang.org/pkg/crypto/sha256/ should work too

Reply:
from their benchmark, it said that it is faster then standard go, also it is the one that used by our in memory cache library